src/components/Clients/ClientListTable.jsx

```diff
@@ -0,0 +1,86 @@
+// React Imports
+import React, { useContext } from 'react';
+// Material UI Imports
+import { styled, useTheme } from '@mui/material/styles';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell, { tableCellClasses } from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+// Custom Hook Imports
+import { useStatusNotification } from '../../hooks';
+// Context Imports
+import { UserListContext } from '../../contexts';
+// Component Imports
+import { StatusNotification } from '../Notification';
+import ClientListTableRow from './ClientListTableRow';
+
+/**
+ * ClientListTable Component - Component that generates table of clients from data within ClientList
+ *
+ * @memberof Clients
+ * @name ClientListTable
+ */
+
+const ClientListTable = ({ statusType, defaultMessage }) => {
+  const { state, dispatch } = useStatusNotification();
+  const { userListObject } = useContext(UserListContext);
+
+  // ===== MAKE CHANGES HERE FOR TABLE HEADER / COLUMN TITLES =====
+  const columnTitlesArray = ['Select', 'Client', 'WebID', 'Last Activity', 'Pin', 'Delete'];
```

Comment:
Think we can move this list outside of the component so it doesn't have to be called each time the component is rendered